portability-spi-transfer/src/main/java/org/datatransferproject/spi/transfer/provider/converter/MediaToPhotoConversionImporter.java

```diff
@@ -23,6 +23,7 @@
  */
 // TODO(#1065) fix primitives-obession causing us to key Providers on "PHOTOS" string rather
 // than underlying file types.
+@Deprecated
```

Comment:
everywhere you add this annotation, can you comment also what the recommended alternative is, as an inline comment? eg:

```
@Deprecated // prefer FooBar#baz
```